### DISCUSSION TITLE
Distributed Mode Help!

### DISCUSSION THREAD

**Discussion by ZethCon07** [Security-Onion-Solutions/securityonion#10848] [General]
I am new to SO and I am trying to see if there are instructions or videos to setting up forward, search, and master nodes. Attempting to setup 3 separate networks, two with forwarding nodes and one network holding the Master and Search nodes. I have been unsuccessful searching over documentation or videos for a walk through to do so. Please advise and thank you!

**Comment by cm-ops** [comment]
Have you looked at https://docs.securityonion.net/en/2.3/configuration.html#production-server-distributed-deployment?
